paymentsheet/src/main/java/com/stripe/android/paymentsheet/paymentdatacollection/polling/PollingActivity.kt

```diff
@@ -80,7 +81,16 @@ internal class PollingActivity : AppCompatActivity() {
                     state = state,
                     onDismissed = { /* Not applicable here */ },
                 ) {
-                    PollingScreen(viewModel)
+                    val qrCodeUrl = args.qrCodeUrl
+                    if (uiState.shouldShowQrCode && qrCodeUrl != null) {
+                        QrCodeWebView(
```

Comment:
Are we doing the right thing on configuration change/process death here? (showing the correct screen)